opentelemetry-jaeger/src/exporter/mod.rs

```diff
@@ -245,6 +247,12 @@ impl PipelineBuilder {
         self
     }
 
+    /// Config whether to auto split batch if packet size is too large.
```

Comment:
Small nit. Can we add some notes on how it works? For example, if one span is too large, will other smaller spans in the same batch being exported. Thanks in advance.

Reply:
I'm not native speaker, and there might be some grammatical errors, feel free to push commit to my branch.

Reply:
Sorry, that was supposed to be a question. From what I understand, if one span is too large. Other spans may or may not get exported, depends on the index of the large span. Any span that comes after the large span will be discarded as we will error out on the large span.